src/controllers/action-queue/mock.js

```diff
@@ -0,0 +1,349 @@
+          },
+          broadcastTxs: mockBroadcastTxs(wallet)
+        }
+      })
+    }
+
+    case 'loan-borrow': {
+      const { walletId } = actionOp
+      const wallet = await account.waitForCurrencyWallet(walletId)
+
+      return mockExecutableAction(account, (): ExecutionOutput => {
+        return {
+          effect: {
+            type: 'tx-confs',
+            txId: mockDelayTimestamp(2000, 5000),
+            walletId: walletId,
+            confirmations: 1
+          },
+          broadcastTxs: mockBroadcastTxs(wallet)
+        }
+      })
+    }
+    case 'loan-deposit': {
+      const { walletId } = actionOp
+      const wallet = await account.waitForCurrencyWallet(walletId)
+
+      return mockExecutableAction(account, (): ExecutionOutput => {
+        return {
+          effect: {
+            type: 'tx-confs',
+            txId: mockDelayTimestamp(3000, 5000),
+            walletId: walletId,
+            confirmations: 1
+          },
+          broadcastTxs: mockBroadcastTxs(wallet)
+        }
+      })
+    }
+    case 'loan-repay': {
+      const { walletId } = actionOp
+      const wallet = await account.waitForCurrencyWallet(walletId)
+
+      return mockExecutableAction(account, (): ExecutionOutput => {
+        return {
+          effect: {
+            type: 'tx-confs',
+            txId: mockDelayTimestamp(2000, 5000),
+            walletId: walletId,
+            confirmations: 1
+          },
+          broadcastTxs: mockBroadcastTxs(wallet)
+        }
+      })
+    }
+    case 'loan-withdraw': {
+      const { walletId } = actionOp
+      const wallet = await account.waitForCurrencyWallet(walletId)
+
+      return mockExecutableAction(account, (): ExecutionOutput => {
+        return {
+          effect: {
+            type: 'tx-confs',
+            txId: mockDelayTimestamp(3000, 5000),
+            walletId: walletId,
+            confirmations: 1
+          },
+          broadcastTxs: mockBroadcastTxs(wallet)
+        }
+      })
+    }
+    case 'swap': {
+      const { toWalletId, toTokenId, fromWalletId: walletId } = actionOp
+      const wallet = await account.waitForCurrencyWallet(walletId)
+
+      return mockExecutableAction(account, (): ExecutionOutput => {
+        return {
+          effect: {
+            type: 'address-balance',
+            address: mockDelayTimestamp(5000, 10000),
+            walletId: toWalletId,
+            tokenId: toTokenId
+          },
+          broadcastTxs: mockBroadcastTxs(wallet)
+        }
+      })
+    }
+
+    case 'broadcast-tx': {
+      throw new Error(`No implementation for action type ${actionOp.type}`)
+    }
+    case 'done': {
+      throw new Error(`No implementation for action type ${actionOp.type}`)
+    }
+    case 'fiat-buy': {
+      throw new Error(`No implementation for action type ${actionOp.type}`)
+    }
+    case 'noop': {
+      throw new Error(`No implementation for action type ${actionOp.type}`)
+    }
+    case 'unixtime': {
```

Comment:
Didn't you just delete `unixtime` in the previous commit?

Reply:
Yes, I removed those after this commit then rebased. I thought I caught this in the rebase. Thanks for noticing.